.github/workflows/ci-maven.yaml

```diff
@@ -75,22 +75,21 @@ jobs:
           java-version: 11
           distribution: temurin
           cache: maven
-      - name: Install modules outside gapic-generator-java
+      - name: Install all modules using Java 11
         shell: bash
         run: |
-          mvn -B -ntp install --projects '!gapic-generator-java' \
-              -Dcheckstyle.skip -Dfmt.skip -DskipTests
+          mvn -V -B -ntp clean install -DskipTests
       - uses: actions/setup-java@v3
         with:
           java-version: 8
           distribution: temurin
           cache: maven
       - run: java -version
-      - name: Run test only for gapic-generator-java using Java 8
+      - name: Run tests in Java 8 with the source compiled in Java 11 for gapic-generator-java
```

Comment:
@blakeli0 @burkedavison @meltsufin This test will now ensure that gapic-generator-java submodule is compiled with Java 11 and the sources+binaries generated are Java 8 compatible. Since we are checking this here only, there shouldn't be a need to handle this in the release job. 
Am I thinking clearly here?  
@suztomo ensured this kind of compatibility in gax (https://github.com/googleapis/gax-java/blob/main/.kokoro/build.sh) when it was a split-repo.